portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,111 @@
+<!-- TODO: Remove optional tags -->
 <!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jay Saleh - Projects & Blog</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <header id="header">
+      <h1>Jay Saleh</h1>
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Background</a></li>
+        <li><a href="#" id="selected">Projects & Blog</a></li>
+      </ul>
+    </header>
+    <div id="blog-body">
+      <div class="post">
+        <div class="post-img">
+          <img src="./images/mexico-trip.jpg" alt="Mexico Picture">
+        </div>  
+        <div class="post-text">
+          <h1>Mexico Trip</h1>
+          <p>
+            After wrapping up freshman year, my high school friends and I reunited in Puerto Vallarta, Mexico
+            for about 9 days. We spent most of our time in Mexico relaxing on the beach, exploring nearby cities,
+            trying different foods, and snorkeling. My favorite restaurant during this trip was the
+            <b><i><a href="https://bit.ly/3fU8Dr6" target="_blank" class="link">Sonora Grill Prime.</a></i></b>
+            The bean and cheese dip and the atmosphere in the restaurant were the highlight. This is one of my
+            favorite trips I’ve been on and I will definitely remember the time spent with my friends.
+          </p>
+        </div>
+      </div>
+      <hr>
+      <div class="post">
+        <div class="post-text">
+          <h1>Toilet Bowl Cliff Jumping</h1>
+          <p>
+            Following my high school graduation, I went on a weekend trip with my friends to Santa Cruz, California.
+            During our weekend trip, one of our friends had the brilliant idea to go cliff jumping at the 
+            <i><b><a href="https://bit.ly/31jEZYq" target="_blank" class="link">Toilet Bowl.</a></b></i> 
```

Comment:
nit: the period should not be included in the link

Reply:
done

Reply:
done